x-pack/legacy/plugins/siem/public/utils/route/spy_routes.tsx

```diff
@@ -56,6 +56,7 @@ export const SpyRouteComponent = memo<SpyRouteProps & { location: H.Location }>(
               detailName,
               tabName,
               search,
+              state,
```

Comment:
@XavierM if I didn't have this, the state I set in `x-pack/legacy/plugins/siem/public/pages/case/components/case_view/index.tsx` doesn't get added